scripts/deploy_limited_allowed_recipients.py

```diff
@@ -0,0 +1,149 @@
+from brownie import chain, network
+
+from utils.config import (
+    get_env,
+    get_is_live,
+    get_deployer_account,
+    prompt_bool,
+    get_network_name,
+)
+from utils import (
+    deployment,
+    lido,
+    deployed_easy_track,
+    log
+)
+
+from brownie import (
+    AllowedRecipientsRegistry,
+    AddAllowedRecipient,
+    RemoveAllowedRecipient,
+    TopUpAllowedRecipients
+)
+
+def main():
+    network_name = get_network_name()
+
+    contracts = lido.contracts(network=network_name)
+    et_contracts = deployed_easy_track.contracts(network=network_name)
+    deployer = get_deployer_account(get_is_live(), network=network_name)
+
+    easy_track = et_contracts.easy_track
+    evm_script_executor = et_contracts.evm_script_executor
+
+    # address allowed to create motions to add, remove or top up allowed recipients
+    committee_multisig = get_env("COMMITTEE_MULTISIG")
+
+    bokkyPooBahsDateTimeContract = get_env("BOOKYPOOBAH_DATETIME_CONTRACT")
```

Comment:
Simple check if we have all env vars set up?

Reply:
no longer relevant 😄